index.html

```diff
@@ -70,10 +83,13 @@ <h3 class="s3h3">Multi-Post Stories</h3>
             </li>
           </ul>
 
-          <button class="btn1">See Project</button>
```

Comment:
- [ ] Also as the previous reviewer asked, it would be better if you implement the style of the button when pressed as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/166074380-c489d449-19c8-400a-9233-6a3296f2f816.png)